packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -43,25 +46,30 @@ export class DiagnosticFixPlugin extends Plugin {
         continue;
       }
 
-      const fix = codefixes.getCodeFixes(diagnostic);
-      if (!fix) {
+      const fixes = codefixes.getCodeFixes(diagnostic);
+
+      if (fixes.length === 0) {
         continue;
-      } else {
-        for (const fileTextChange of fix.changes) {
-          let text = this.service.getFileText(fileTextChange.fileName);
+      }
 
-          const textChanges = normalizeTextChanges([...fileTextChange.textChanges]);
-          for (const textChange of textChanges) {
-            text = applyTextChange(text, textChange);
-          }
+      // Use the first available codefix in automatic mode,
+      // TODO: User should be able to choose one of the fixes form this list in interactive mode
+      const fix = fixes.shift()!;
```

Comment:
@lynchbomb we will be able to interact with users in this place to allow them to choose which fix to apply :)